packages/table/src/Table.story.tsx

```diff
@@ -1,363 +1,838 @@
+            {table.getHeaderGroups().map((headerGroup: HeaderGroup<Person>) => (
+              <HeaderRow key={headerGroup.id}>
+                {headerGroup.headers.map(header => {
+                  return (
+                    <HeaderCell key={header.id} header={header}>
+                      {flexRender(
+                        header.column.columnDef.header,
+                        header.getContext(),
+                      )}
+                    </HeaderCell>
+                  );
+                })}
+              </HeaderRow>
+            ))}
+          </TableHead>
+          <TableBody>
+            {rows.map((row: LeafyGreenTableRow<Person>) => {
+              return (
+                <Row key={row.id} row={row}>
+                  {row.getVisibleCells().map(cell => {
+                    return (
+                      <Cell key={cell.id}>
+                        {flexRender(
+                          cell.column.columnDef.cell,
+                          cell.getContext(),
+                        )}
+                      </Cell>
+                    );
+                  })}
+                </Row>
+              );
+            })}
+          </TableBody>
+        </Table>
+      </TableContainer>
+    </>
+  );
 };
 
-export const NestedRowsWithLongContent = ({
-  withHeaders,
-  ...args
-}: TableArgs<any>) => (
-  <Table
-    {...args}
-    data={[
-      {
-        title: 'People',
-        people: [
-          {
-            name: 'Nulla vitae elit libero, a pharetra augue. Sed posuere consectetur est at lobortis.',
-            age: 19,
-            color: 'blue',
-            location: 'bedford',
-            rand: Math.random(),
-          },
-          ...defaultData,
-        ],
-      },
-      {
-        title: 'Average',
-        age: (
-          defaultData.reduce((sum, { age }) => sum + age, 0) /
-          defaultData.length
-        ).toFixed(2),
-      },
-    ]}
-    columns={
-      <HeaderRow>
-        <TableHeader key="name" label="Name" dataType="string" />
-        <TableHeader key="age" label="Age" dataType="number" />
-        <TableHeader label="Color" dataType="string" key="color" />
-        <TableHeader key="location" label="Location" />
-      </HeaderRow>
-    }
-    style={{ maxWidth: '400px ' }}
-  >
-    {({ datum }: { datum: any }) => (
-      <Row key={datum.title}>
-        <Cell isHeader={withHeaders}>{datum.title}</Cell>
-
-        {datum.people ? (
-          datum.people.map((person: any) => (
-            <Row key={person.name}>
-              <Cell isHeader={withHeaders}>{person.name}</Cell>
-              <Cell>{person.age}</Cell>
-              <Cell>{person.color}</Cell>
-              <Cell>{person.location}</Cell>
-            </Row>
-          ))
-        ) : (
-          <Cell>{datum.age}</Cell>
-        )}
-      </Row>
-    )}
-  </Table>
-);
-NestedRowsWithLongContent.args = {
-  withHeaders: true,
+export const KitchenSink: ComponentStory<typeof Table> = args => {
```

Comment:
What happens with content that overflows horizontally as this table demonstrates?

Reply:
There's an `overflow: hidden` on the td which you can see with the ID field on other stories. In this story, it should probably not overflow so I'm going to change that.

Reply:
We need some design pattern around what happens with the Table extends past the edge of the viewport

Reply:
Going to keep the `overflow: hidden` on the Cell, and ask the developers to implement ellipses behavior on their end because adding `text-overflow: ellipses` requires us have a `width: _px` set on the div, which we can't do because we don't know the width of the Cell